src/core/config_opt_mod.F90

```diff
@@ -74,8 +74,10 @@ MODULE Config_Opt_Mod
       ! Dust Process
       LOGICAL                     :: dust_activate
       INTEGER                     :: dust_scheme
-      INTEGER                     :: dust_fengsha_drag_opt  ! Fengsha Option for drag Paramertization (1 MB95; 2) Input Value)
-      INTEGER                     :: dust_fengsha_moist_opt ! Fengsha Option for moisture Paramertization (1 Fecan; 2 Shao)
+      INTEGER                     :: dust_drag_opt  ! Fengsha Option for drag Paramertization (1 MB95; 2) Input Value)
```

Comment:
```suggestion
      INTEGER                     :: dust_drag_opt  ! Fengsha Option for drag Parameterization (1 MB95; 2 Input Value)
```

Reply:
Also note `!<` can be used instead of `!` for Doxygen to recognize this as a parameter description